eng/packages/http-client-csharp/generator/Azure.Generator/src/AzureTypeFactory.cs

```diff
@@ -52,7 +52,6 @@ public class AzureTypeFactory : ScmTypeFactory
         protected internal virtual IReadOnlyList<CSharpProjectWriter.CSProjDependencyPackage> AzureDependencyPackages =>
             [
                 new("Azure.Core"),
-                new("System.ClientModel"),
                 new("System.Text.Json")
```

Comment:
Based on https://github.com/Azure/azure-sdk-for-net/pull/49161, we should probably also remove System.Text.Json.